extensions/ql-vscode/src/log-insights/summary-language-support.ts

```diff
@@ -76,11 +77,16 @@ export class SummaryLanguageSupport extends DisposableObject {
       const mapPath = document.uri.fsPath + '.map';
 
       try {
-        const sourceMapText = await fs.promises.readFile(mapPath, 'utf-8');
+        const sourceMapText = await fs.readFile(mapPath, 'utf-8');
         const rawMap: RawSourceMap = JSON.parse(sourceMapText);
         this.sourceMap = await new SourceMapConsumer(rawMap);
-      } catch {
-        // Error reading sourcemap. Pretend there was no sourcemap
+      } catch (e: unknown) {
+        // Error reading sourcemap. Pretend there was no sourcemap.
+        if (e instanceof Error) {
+          void logger.log(`Error reading sourcemap file '${mapPath}': ${e.message}`);
+        } else {
+          void logger.log(`Error reading sourcemap file '${mapPath}'`);
+        }
```

Comment:
Minor: There is a helper function that will make this simpler. 99.9% of the time (maybe 100% if the time here) this will be an instance of `Error`. See the `getErrorMessage` in `helpers.ts`.

Reply:
Thanks. Done.